src/markdown.cpp

```diff
@@ -1145,7 +1145,6 @@ int Markdown::processLink(const char *data,int offset,int size)
   {
     whiteSpace = true;
     i++;
-    nl++;
```

Comment:
I guess this is removed because the whitespace here is not a part of the output (i.e. it is a separator between the `[..]` and `(..)` parts). But then lines 1152 and 1153 also do not make much sense, as `nl` is initialized to 0 at line 1133 and then not modified anymore until it is set to 0 again on line 1153.

Reply:
Indeed they don't make much sense anymore now the `nl++` is gone, adjusted.